docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -63,8 +76,8 @@
 			<p><img alt="" height="225" src="media/tv3uicontrolsample.png" width="400" /></p>
 			</td>
 			<td>
-			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/UIControlSample" target="_blank"><strong>UIControl Sample</strong></a></p>
-			<p>The sample application demonstrates how NUI views work on TV.<br />
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/TV/UIControlSample" target="_blank"><strong>UIControl Sample (NUI)</strong></a></p>
+			<p>This sample application demonstrates how NUI views work on TV.<br />
```

Comment:
NUI means Natural User Interface.. C# toolkit binding DALi...

Is it correct ?
UIControl Sample (Natural User Interface (NUI))

Reply:
I am waiting for your response regarding this issue....

(NUI) is added for TV sample applications which are made by NUI, instead of using Xamarin.Forms.

Reply:
I think .. no need to use the full name of NUI... 
In the C# API Reference, only NUI exists, Natural User Interface (NUI) is not used at any places...

Reply:
ok @sgchoi5 , it was just a suggestion. 
Sorry was reviewing another file.